indico/web/client/js/react/components/ICSCalendarLink.module.scss

```diff
@@ -9,7 +9,39 @@
   height: 100%;
 }
 
-.centered {
+.export-header:global(.ui.header) {
   display: flex;
   justify-content: center;
+  margin-bottom: 0;
+}
+
+.export-option {
+  display: block;
+  text-align: center;
+  margin: 1rem 0 0.5rem 0;
+}
+
+.export-option::before,
+.export-option::after {
+  box-shadow: 1px 1px 1px #f9f9f9;
+  background-color: #dfdfdf;
+  content: '';
+  display: inline-block;
+  height: 1px;
+  position: relative;
+  vertical-align: middle;
+  width: 10%;
+  margin: 0 0.5rem;
+}
+
+.download-button:global(.ui.icon.button) {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  height: 2.125rem;
```

Comment:
Where's this number coming from?

Reply:
Comes from the height of SUI input component. The idea is for both buttons (this and the copy one on top of him) to have the same size.